### PR TITLE
Completed first pass of .adoc files and revised template

### PR DESCRIPTION
**Pull request by DelfinGala**
*Description of changes:*
Updated partner_editable content and eks-snyk.template.yaml. Note that there are embedded //TODO comments.

It seems that large portions of text were entered into the boilerplate section of the repo. All of those items should be relocated to partner_editable. I made changes to several boilerplate files, but I won't commit/push them until we sort out this issue.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by jaymccon — CHANGES_REQUESTED** [review]
I've removed the redundant files that were the source of most of the `TODO`. Added review comments for everything else.

File: templates/eks-snyk.template.yaml [hunk 1]
```diff
@@ -15,22 +15,22 @@ Metadata:
...
   KubeClusterName:
     Type: String
-    Description: Name of the EKS cluster to deploy snyk into.
+    Description: Name of the EKS cluster to deploy Snyk into.
   SnykIntegrationId:
     Type: String
     AllowedPattern: '[a-z0-9-]{36}'
     MinLength: 36
-    Description: 'Snyk kubernetes integration ID. Must be obtained from the Snyk console. For more information see https://support.snyk.io/hc/en-us/articles/360003916158-Install-the-Snyk-controller-with-Helm .'
+    Description: 'Snyk Kubernetes Integration ID. This must be obtained from the Snyk console. For more information, see https://support.snyk.io/hc/en-us/articles/360003916158-Install-the-Snyk-controller-with-Helm .'
```
[the remark this answers is not in the document]
Comment (DelfinGala) [reply templates/eks-snyk.template.yaml:29]: Kubernetes should definitely be capitalized. I'm not sure why I capitalized "integration." Seems like a mistake on my part.

File: docs/partner_editable/_settings.adoc [hunk 2]
```diff
@@ -1,15 +1,16 @@
 :quickstart-project-name: quickstart-eks-snyk
-:partner-product-name: Snyk controller for Amazon EKS
+:partner-product-name: Snyk Controller for Amazon EKS on the AWS Cloud
```
[the remark this answers is not in the document]
Comment (DelfinGala) [reply docs/partner_editable/_settings.adoc:2]: True, but this is the value that goes into the title, where it should be capitalized.

File: docs/partner_editable/architecture.adoc [hunk 3]
```diff
@@ -1,5 +1,7 @@
 
-* Kubernetes namespace for Snyk
-* Kubernetes secret containing Skyk integration ID and docker config json.
-* Snyk monitor pod
+* A Kubernetes namespace for Snyk.
+* A Kubernetes secret that contains a Snyk Integration ID and docker config JSON file.
+* A Snyk monitor pod.
+
+//TODO This list should capture everything in the diagram. It's currently missing most of the items I see in the diagram.
```
[the remark this answers is not in the document]
Comment (DelfinGala) [reply docs/partner_editable/architecture.adoc:6]: Fair enough.

File: docs/partner_editable/licenses.adoc [hunk 4]
```diff
@@ -1 +1 @@
-This Quick Start requires a license for Snyk, the license costs and the instructions to obtain a license are available on https://aws.amazon.com/marketplace/pp/B085VGM85Q?qid=1590170928622&sr=0-1&ref_=srh_res_product_title[Snyk's AWS marketplace page^].
+This Quick Start requires a Snyk license. For more information, see https://aws.amazon.com/marketplace/pp/B085VGM85Q?qid=1590170928622&sr=0-1&ref_=srh_res_product_title[Snyk: Developer-First Security^].
```
[the remark this answers is not in the document]
Comment (DelfinGala) [reply docs/partner_editable/licenses.adoc:1]: Why is it necessary to call out AWS Marketplace?

File: templates/eks-snyk.template.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (DelfinGala) [reply templates/eks-snyk.template.yaml:29]: Kubernetes should be capitalized. "Integration" can be set in lowercase.

**Pull request merged**